.env-dist

```diff
@@ -18,7 +18,7 @@ OAUTH_TOKEN_URI="https://oauth-stable.dev.lcip.org/v1/token"
 
 HIBP_API_ROOT="https://stage.haveibeenpwned.com/api/v2/"
 HIBP_API_TOKEN=""
-HIBP_STAGE_API_ROOT="https://stage-api.haveibeenpwned.com"
+HIBP_STAGE_API_ROOT="https://api.haveibeenpwned.com"
```

Comment:
let's keep this stage-api in .env-dist; we should use the HIBP_API_ROOT for production and we can fix this up in another PR

Reply:
agreed!

Reply:
I'm confused... It almost looks reversed.
The `HIBP_API_ROOT` references stage.haveibeenowned.com and now the `HIBP_STAGE_API_ROOT` is pointing at api.haveibeenpwned.com, which looks more production-y.